veniq/dataset_collection/augmentation.py

```diff
@@ -75,6 +80,40 @@ def method_body_lines(method_node: ASTNode, file_path: Path) -> Tuple[int, int]:
     return start_line, end_line
 
 
+def check_nesting_statements(
+        method_invoked: ASTNode
+) -> bool:
+    """
+    Check that the considered method invocation is not
+    at the same line as prohibited statements.
+    """
+    prohibited_statements = [
+        ASTNodeType.IF_STATEMENT,
+        ASTNodeType.WHILE_STATEMENT,
+        ASTNodeType.FOR_STATEMENT,
+        ASTNodeType.SYNCHRONIZED_STATEMENT,
+        ASTNodeType.CATCH_CLAUSE,
+        ASTNodeType.SUPER_CONSTRUCTOR_INVOCATION,
+        ASTNodeType.TRY_STATEMENT
+    ]
+    if method_invoked.parent is not None:
```

Comment:
Maybe it is worth to join everything into a single `if`. If you gonna do this, you better add some boolean variables with meaningful name.